pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<parent>
+  <groupId>org.pentaho</groupId>
+  <artifactId>pentaho-ce-jar-parent-pom</artifactId>
+  <version>2.0.4</version>
+</parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>pentaho</groupId>
+  <artifactId>pentaho-palo-core</artifactId>
+  <version>8.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <scm>
+    <developerConnection>scm:git:git@github.com:pentaho/pdi-palo-core.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+  <dependencies>  
+  <dependency>
+        <groupId>pentaho-kettle</groupId>
+        <artifactId>kettle-core</artifactId>
+        <version>8.0-SNAPSHOT</version>

```

Comment:
dependencies versions should be defined as properties, in this special case define a `kettle.version` property